src/components/header/index.tsx

```diff
@@ -12,20 +13,23 @@ export const Header = () => {
 
   return (
     <nav className="z-100 flex items-center justify-between px-6">
-      <div className="flex items-center gap-12">
+      <div className="flex items-center gap-8">
         <Logo />
         <Navigation />
       </div>
-      <div className="flex gap-5">
+      <div className="hidden gap-5 lg:flex">
         {
           accounts.length > 0
             ? <button type="button" onClick={disconnectWallet}>Disconnect Wallet</button>
             : <button type="button" onClick={connectWallet}>Connect Wallet</button>
         }
-
         <ChainSelectButton />
         <ToggleTheme />
       </div>
+      <div className="flex lg:hidden">
```

Comment:
Check breakpoints, for some screen sizes both burger and navigation items are present

Reply:
Done